core/src/main/scala/filodb.core/downsample/DownsampledTimeSeriesStore.scala

```diff
@@ -200,5 +200,6 @@ extends MemStore with StrictLogging {
   override def topKCardinality(ref: DatasetRef,
                                shards: Seq[Int],
                                shardKeyPrefix: scala.Seq[String],
-                               k: Int): scala.Seq[CardinalityRecord] = ???
+                               k: Int,
+                               totalNotActive: Boolean): scala.Seq[CardinalityRecord] = ???
```

Comment:
This was an attempt to avoid an enum. Ideally, it would be an enum with `active` and `total` as values, but I didn't want to do an enum with just two values. Boolean was simpler. And I wanted both enum values in the name for easier understanding.

If you guys think, having an enum is much better, I can do that.

Reply:
For now, renamed `totalNotActive` to `addInactive`